capstone/src/main/java/com/google/step/data/Restaurant.java

```diff
@@ -66,6 +66,62 @@ public static Restaurant fromEntity(Entity entity) {
     return new Restaurant(restaurantKey, name, location, story, cuisine, phone, website, status);
   }
 
+  /**
+   * @return long restaurantKey, unique identifier for the restaurant
+   */
+  public long getRestaurantKey() {
+    return this.restaurantKey;
+  }
+
+  /**
+   * @return String name, name of the restaurant
+   */
+  public String getName() {
+    return this.name;
+  }
+
+  /**
+   * @return GeoPt location, containing latitude and longitude of the restaurant
+   */
+  public GeoPt getLocation() {
+    return this.location;
+  }
+
+  /**
+   * @return String story, personal story of the restaurant
+   */
+  public String getStory() {
+    return this.story;
+  }
+
+  /**
+   * @return List<String> cuisine, list of cuisines for the restaurant
+   */
+  public List<String> getCuisine() {
+    return this.cuisine;
```

Comment:
You should return a copy of the cuisine, otherwise clients of this class can "add" to it and mess with its internals.

Or, since everything is meant to be final, use an immutable list as the internal representation.